### DISCUSSION TITLE
Highlight field and value in an insert command

### DISCUSSION THREAD

**Discussion by drusso86** [dbeaver/dbeaver#22211] [Ideas]
Hello, I would like to suggest a new feature for DBeaver. My suggestion is that the field and its respective value be highlighted in an INSERT command as the cursor passes through the field or value. This would help avoid typing errors and make editing INSERT commands easier and more intuitive.

Thank you!

**Comment by E1izabeth** [comment]
It's a good idea. Could you create a feature request?

**Reply by drusso86** [reply]
Done. I don't know if I opened it correctly, but issue #22522 is there.

**Reply by E1izabeth** [reply]
Thank you
